src/main/java/com/google/impactdashboard/api_utilities/LogRetriever.java

```diff
@@ -0,0 +1,15 @@
+package com.google.impactdashboard.api_utilities;
+
+import com.google.cloud.logging.v2.LoggingClient;
+
```

Comment:
super nit: no newlines between import statements https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

Reply:
Ops I think that was a setting predefined in intellij I'll fix that